lib/gamco/builder.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Gamco.Builder do
+  import Phoenix.Component
+
+  @gtag_manager_url "https://www.googletagmanager.com/gtag/js"
+
+  def javascript_tags(tag_id, opts \\ []) do
+    data =
+      opts
+      |> Enum.into(%{})
+      |> Jason.encode!()
+
+    assigns =
+      %{}
+      |> Map.put(:data, data)
+      |> Map.put(:tag_id, tag_id)
+      |> Map.put(:gtag_manager_url, @gtag_manager_url)
+
+    ~H"""
```

Comment:
even for these kind of stuff I prefer heex files instead of adding it to string

Reply:
I rather inline html because this is a pretty simple library with small pieces of html and IMO there is not need to add heex files.

Reply:
I would argue that even if is a simple library having the heex files allows you to organize and better maintain the code

Reply:
Inline html using sigils it is a very good practice, there is nothing wrong with this. Even new phoenix applications come with this as a pattern and are very maintainable.